modules/user_accounts/jsx/userAccountsIndex.js

```diff
@@ -0,0 +1,178 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import Loader from 'Loader';
+import FilterableDataTable from 'FilterableDataTable';
+
+class UserAccountsIndex extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: {},
+      error: false,
+      isLoaded: false,
+    };
+
+    this.fetchData = this.fetchData.bind(this);
+    this.formatColumn = this.formatColumn.bind(this);
+  }
+
+  componentDidMount() {
+    this.fetchData()
+      .then(() => this.setState({isLoaded: true}));
+  }
+
+  /**
+   * Retrieve data from the provided URL and save it in state
+   * Additionally add hiddenHeaders to global loris variable
+   * for easy access by columnFormatter.
+   *
+   * @return {object}
+   */
+  fetchData() {
+    return fetch(this.props.dataURL, {credentials: 'same-origin'})
+      .then((resp) => resp.json())
+      .then((data) => {
+        // Convert concatenated string of sites to array
+        for (let key in data.Data) {
+          if (data.Data.hasOwnProperty(key)) {
```

Comment:
Since i believe `data.Data` is an array, you can do something like: 
```
data.Data = data.Data.map((row) => {
  row[0] = row[0].split('; '); // sites are the first element of the row array
  return row;
}
```
This avoids the need for the `hasOwnProperty` check, and is more in line with how javascript deals with arrays.